<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1920</width>
     <height>1000</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1000</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
    <item>
     <widget class="QGraphicsView" name="GraphicsView"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QGroupBox" name="RotateModelBox">
         <property name="title">
          <string>Поворот</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>X</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Y</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Z</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QDoubleSpinBox" name="RotateModelax">
               <property name="minimum">
                <double>-100000000000000004764729344.000000000000000</double>
               </property>
               <property name="maximum">
                <double>9999999999999999455752309870428160.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="RotateModelay">
               <property name="minimum">
                <double>-100000000000000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000000000000000000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QDoubleSpinBox" name="RotateModelaz">
               <property name="minimum">
                <double>-10000000000000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000000000000000.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="RotateModelButton">
            <property name="text">
             <string>Повернуть</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="RotateAllModelsButton">
            <property name="text">
             <string>Повернуть все</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QGroupBox" name="ModelsManagmentBox">
         <property name="title">
          <string>Модели</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QComboBox" name="ModelsComboBox"/>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="DeleteModelButton">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="LoadModelButton">
            <property name="text">
             <string>Загрузить</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="MoveModelBox">
         <property name="title">
          <string>Перемещение</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1,1">
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>X</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Y</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Z</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QDoubleSpinBox" name="MoveModeldx">
               <property name="minimum">
                <double>-10000000000000000000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>10000000000000000000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="MoveModeldy">
               <property name="minimum">
                <double>-1000000000000000000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>99999999999999991611392.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QDoubleSpinBox" name="MoveModeldz">
               <property name="minimum">
                <double>-10000000000000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>10000000000000000000.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="MoveModelButton">
            <property name="text">
             <string>Переместить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="MoveAllModelsButton">
            <property name="text">
             <string>Переместить все</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="ScaleModelBox">
         <property name="title">
          <string>Масштабирование</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QWidget" name="widget_4" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="1">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Y</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Z</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>X</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QDoubleSpinBox" name="ScaleModelkx">
               <property name="minimum">
                <double>-1000000000000000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>10000000000000000.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="ScaleModelky">
               <property name="minimum">
                <double>-100000000000000000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>100000000000000000000.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QDoubleSpinBox" name="ScaleModelkz">
               <property name="minimum">
                <double>-100000000000000004764729344.000000000000000</double>
               </property>
               <property name="maximum">
                <double>999999999999999983222784.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="ScaleModel">
            <property name="text">
             <string>Масштабировать</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="ScaleAllModelsButton">
            <property name="text">
             <string>Масштабировать все</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QGroupBox" name="CameraManagmentBox">
         <property name="title">
          <string>Камеры</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="2">
           <widget class="QPushButton" name="AddCameraButton">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="DeleteCameraButton">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QComboBox" name="CamerasComboBox"/>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QWidget" name="widget_5" native="true">
            <widget class="QPushButton" name="CameraMoveUpButton">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>10</y>
               <width>31</width>
               <height>25</height>
              </rect>
             </property>
             <property name="text">
              <string>↑</string>
             </property>
            </widget>
            <widget class="QPushButton" name="CameraMoveRightButton">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>40</y>
               <width>31</width>
               <height>25</height>
              </rect>
             </property>
             <property name="text">
              <string>→</string>
             </property>
            </widget>
            <widget class="QPushButton" name="CameraMoveLeftButton">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>40</y>
               <width>31</width>
               <height>25</height>
              </rect>
             </property>
             <property name="text">
              <string>←</string>
             </property>
            </widget>
            <widget class="QPushButton" name="CameraMoveDownButton">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>70</y>
               <width>31</width>
               <height>25</height>
              </rect>
             </property>
             <property name="text">
              <string>↓</string>
             </property>
            </widget>
            <widget class="QPushButton" name="CameraRotateLeftButton">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>40</y>
               <width>31</width>
               <height>25</height>
              </rect>
             </property>
             <property name="text">
              <string>↶</string>
             </property>
            </widget>
            <widget class="QPushButton" name="CameraRotateRightButton">
             <property name="geometry">
              <rect>
               <x>190</x>
               <y>40</y>
               <width>31</width>
               <height>25</height>
              </rect>
             </property>
             <property name="text">
              <string>↷</string>
             </property>
            </widget>
            <widget class="QPushButton" name="CameraRotateUpButton">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>10</y>
               <width>31</width>
               <height>25</height>
              </rect>
             </property>
             <property name="text">
              <string>↑</string>
             </property>
            </widget>
            <widget class="QPushButton" name="CameraRotateeDownButton">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>70</y>
               <width>31</width>
               <height>25</height>
              </rect>
             </property>
             <property name="text">
              <string>↓</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
